ltdconveyor/__init__.py

```diff
@@ -1,6 +1,8 @@
 import pkg_resources
 
-from .s3upload import upload_dir, upload_file, upload_object, ObjectManager  # noqa: F401,E501
+from .s3upload import (upload_dir, upload_file, upload_object,  # noqa: F401
+                       create_dir_redirect_object,  # noqa: F401
+                       ObjectManager)  # noqa: F401
```

Comment:
I didn't know you could do the parens trick.  Nice way to get multiple lines without needing a continuation character.